lib/metrics/eventLoopLag.js

```diff
@@ -16,7 +17,8 @@ module.exports = registry => {
 	const gauge = new Gauge({
 		name: NODEJS_EVENTLOOP_LAG,
 		help: 'Lag of event loop in seconds.',
-		registers: registry ? [registry] : undefined
+		registers: registry ? [registry] : undefined,
+		aggregator: AGGREGATORS.AVERAGE
```

Comment:
Everything defaults to `sum` via code in the `Gauge`/`Histogram`/`Summary`/`Counter` classes ([example](https://github.com/siimon/prom-client/pull/147/files#diff-565b1b7426b6a8327b446f794fbd5d5fR73)), but there are three metrics with non-default methods:
* `nodejs_version_info` uses the first value (all values will be the same)
* `process_starttime_seconds` is omitted (all values will be slightly different -- open to suggestions)
* `nodejs_eventloop_lag` averages